GitCommands/Git/GitCommandHelpers.cs

```diff
@@ -378,45 +379,40 @@ private static string SubmoduleUpdateCommand(string name)
 
         public static string SubmoduleSyncCmd(string name)
         {
-            if (string.IsNullOrEmpty(name))
+            var args = new ArgumentBuilder
             {
-                return "submodule sync";
-            }
+                "submodule sync",
```

Comment:
❗️ I would expect an argument build to automatically quote arguments with spaces when necessary, which means I would expect to see this as two arguments instead of one.